contrib/docker/kubernetes-example-backend/README.md

```diff
@@ -0,0 +1,13 @@
+# Example backend Dockerfile
```

Comment:
Is `kubernetes-example-backend` the right name for the directory? This isn't specific to kubernetes. It would be useful for something like Docker Compose too.

Perhaps something like `omnibus-backend`?

Reply:
Firstly, I think (based on our chat with Spotify earlier) that it's important to make clear this is intended as an example/demo so I'd like to keep "example" in there.

"omnibus" means nothing to me; I wouldn't have a clue what's in that folder.

The kubernetes part was an attempt to show why we need another Dockerfile at all. I was aware that it might have more broad usage but I'm not sure how to communicate that succinctly. A few options:
example-backend-without-docker?
example-backend-without-dind?
example-backend?

Reply:
Solid reasoning. Stick with what you have I guess.